extensions/schema-compare/src/utils.ts

```diff
@@ -16,6 +16,19 @@ export interface IPackageInfo {
 	aiKey: string;
 }
 
+export interface ButtonState {
```

Comment:
this can be with the test class since it's only used by the test code

Reply:
Ahh yes. I will move it to the test class. Thanks for pointing that :).